adafruit_thermal_printer/thermal_printer.py

```diff
@@ -129,12 +129,12 @@ def __init__(self, mask: int) -> None:
 
         def __get__(
             self,
-            obj: "ThermalPrinter",
-            objtype: Type["ThermalPrinter"],
+            obj: Optional["ThermalPrinter"],
+            objtype: Optional[Type["ThermalPrinter"]],
```

Comment:
Hi, sorry for being unclear:

`obj` can be either `None` if it's used on the class, or `"ThermalPrinter"` if it's used on an instance.

Either way, `objtype` will be `Type["ThermalPrinter"]` since that' the type of `obj` regardless.

Reply:
Suggestion below which hopefully helps clear things up.

Reply:
yeah, not sure you were unclear.  I went from my notes, which show optional on type as well, but I'm going to accept that it's likely usage / class dependent on when to use it or not.  

fixed and resubmitted.